neural_compressor/torch/quantization/config.py

```diff
@@ -615,6 +617,163 @@ def get_default_teq_config() -> TEQConfig:
     return TEQConfig()
 
 
+######################## AUTOROUND Config ###############################
+@register_config(framework_name=FRAMEWORK_NAME, algo_name=AUTOROUND, priority=PRIORITY_AUTOROUND)
+class AutoRoundConfig(BaseConfig):
```

Comment:
this config includes too many fields and should check which one can be removed and take default value in internal implenentation

Reply:
removed https://github.com/intel/neural-compressor/pull/1647/commits/08b00d51d214a4740f62038113405de6fdbbc0d6